src/env.ts

```diff
@@ -28,11 +28,13 @@ export const env = createEnv({
    * `experimental__runtimeEnv` as well.
    */
   client: {
-    NEXT_PUBLIC_OPENPANEL_CLIENT_ID: z.string().min(1),
+    NEXT_PUBLIC_OPENPANEL_CLIENT_ID: z.string().optional(),
     NEXT_PUBLIC_FACEBOOK_PIXEL_ID: z.string().min(1),
+    NEXT_PUBLIC_OPENPANEL_ENABLED: z.enum(["true", "false"]).default("false").transform((val) => val === "true"),
```

Comment:
we can avoid to having to sync two related variables.

removing the `NEXT_PUBLIC_OPENPANEL_ENABLED` variable and just check if `NEXT_PUBLIC_OPENPANEL_CLIENT_ID` is defined and non-empty